test/utils/server.js

```diff
@@ -32,16 +39,13 @@ function start(_options) {
         console.log('server options changed; restarting');
         stop();
         options = _options;
-        var offline = (options.offline) || false;
-        console.log('starting restbase in ' + (offline ? 'OFFLINE' : 'ONLINE') + ' mode');
-        return restbase({
-            logging: {
-                name: 'restbase-tests',
-                level: 'trace',
-                stream: config.logStream
-            },
-            spec: config.spec
-        }).then(function(server){
+        console.log('starting restbase in '
+                + (options.offline ? 'OFFLINE' : 'ONLINE') + ' mode');
+        config.conf.offline = options.offline || false;
+
+        return runner.run(config.conf)
+        .then(function(servers){
+            var server = servers[0];
```

Comment:
With servisor, ops, service-runner, this becomes really easy to read. Me likes mucho.